db/scripts/init_tables.sql

```diff
@@ -100,3 +100,5 @@ ALTER TABLE domain_totals DROP COLUMN IF EXISTS connected;
 ALTER TABLE domain_totals ADD COLUMN IF NOT EXISTS with_mxs INTEGER DEFAULT 0;
 
 ALTER TABLE domains ADD COLUMN IF NOT EXISTS mta_sts BOOLEAN DEFAULT FALSE;
+
+ALTER TABLE domain_totals RENAME TO aggregated_scans;
```

Comment:
Do we need to `IF NOT EXISTS` here?

Reply:
Oh, yes! We could add a second run of the migration to our travis config to confirm that it's idempotent :D

Reply:
Good idea!! :D i can do that if you haven't already